src/sentinel.c

```diff
@@ -2780,7 +2780,7 @@ void sentinelSendPeriodicCommands(sentinelRedisInstance *ri) {
         SENTINEL_MAX_PENDING_COMMANDS * ri->link->refcount) return;
 
     /* If this is a slave of a master in O_DOWN condition we start sending
-     * it INFO every second, instead of the usual SENTINEL_INFO_PERIOD
+     * it INFO every 100 milliseconds, instead of the usual SENTINEL_INFO_PERIOD
```

Comment:
Should we maybe make this configurable instead, so people can choose the type of tradeoff that they want?